slides.md

```diff
@@ -1427,7 +1427,7 @@ export default async function users(fastify) {
 <div class="dense">
 
 - Let's create an Fastify application using **TypeScript**.
-- We will transpose the application that you did in the [Step 09](#step-9-decorators) to TypeScript
+- We will transpose the application that you did in the [Step 10](/59) to TypeScript
```

Comment:
```suggestion
- We will transpose the application that you did in the [Step 10](#step-10-decorators) to TypeScript
```

Reply:
I tried this approach at first, but it didn't work for me locally, it just added the anchor to url without redirecting to the correct slide

<img width="1443" alt="Screenshot 2023-04-04 at 17 18 11" src="https://user-images.githubusercontent.com/17121337/229910863-208b995f-69a1-472b-b461-9c05e244a6a4.png">

Reply:
`/59` won't work on the github pages link, if we're at `https://nearform.github.io/the-fastify-workshop/80` it will link to `https://nearform.github.io/59`.

`./59` uses the current path instead of the root url and will work.

Reply:
Good catch, github pages are always tricky to work with 😄